packages/muelu/adapters/tpetra/MueLu_ShiftedLaplacian_decl.hpp

```diff
@@ -95,7 +97,7 @@ class ShiftedLaplacian : public BaseClass {
     , Nullspace_("constant")
     , numLevels_(5)
     , coarseGridSize_(100)
-    , omega_(2.0 * M_PI)
+    , omega_(2.0 * Kokkos::numbers::pi_v<double>)
```

Comment:
FYI: C++20 has [`std::number::pi_v<double>`](https://en.cppreference.com/w/cpp/numeric/constants).  But sadly, Trilinos can't use C++20 yet :-(

Reply:
Yeah, that's what I wanted to use first. In a couple of months we can use that.